test/e2e/crypta_test.go

```diff
@@ -0,0 +1,160 @@
+package crypta_test
+
+import (
+	"fmt"
+	"io"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	. "github.com/onsi/gomega/gexec"
+)
+
+func initDaemon() {
+	BeforeAll(func() {
+		command := exec.Command(pathToCrypta, "daemon")
+		daemon, err := Start(command, GinkgoWriter, GinkgoWriter)
+		Ω(err).ShouldNot(HaveOccurred())
+		Ω(daemon).ShouldNot(Exit(0))
+	})
+
+	AfterAll(func() {
+		KillAndWait()
+	})
+}
+
+func setValue(key, val string) {
+	cmd := exec.Command(pathToCrypta, "set", key, val)
+	crypta, err := Start(cmd, GinkgoWriter, GinkgoWriter)
+	Ω(err).ShouldNot(HaveOccurred())
+
+	Eventually(crypta).Should(Exit(0))
+}
+
+func getValue(key string) string {
+	cmd := exec.Command(pathToCrypta, "get", key)
+	crypta, err := Start(cmd, GinkgoWriter, GinkgoWriter)
+	Ω(err).ShouldNot(HaveOccurred())
+
+	Eventually(crypta).Should(Exit(0))
+
+	return string(crypta.Out.Contents())
+}
+
+var _ = Describe("Crypta", func() {
+	Describe("A set value can be retrieved", Ordered, func() {
+		key := "abcd"
+		val := "xyz"
+		otherVal := "zyx"
+
+		initDaemon()
+
+		It("sets a new value", func() {
+			setValue(key, val)
+		})
+
+		It("retrieves the previously written value", func() {
+			Ω(getValue(key)).Should(Equal(val + "\n"))
```

Comment:
Do you think we should return it without `\n`?

Reply:
Hm.. I am not sure how it will integrate into scripts if there is a `\n` appended to the end of the values. Do you know? Else I can also try it out.

Reply:
Not sure. I guess if you use it like `foo=$(crypta get ...)` it does not really matter but probably makes sense to test it

Reply:
I used the following testscript:

```
#!/bin/bash

dist/crypta_linux_amd64_v1/crypta daemon &

dist/crypta_linux_amd64_v1/crypta set xyz abc

foo=$(dist/crypta_linux_amd64_v1/crypta get xyz)

echo "ffff${foo}ffff"
echo $foo
```

and received the following output:

```
ffffabcffff
abc
```

So I would say that the `\n` does not have a negative influence, at least on tool usage in scripts. Do you see any other reason why we should remove it?